### ISSUE TITLE
Export your resume npt working

### ISSUE THREAD

**Issue by SaiAnuragGudi** [AmruthPillai/Reactive-Resume#402]
When I select the single page or the multi page resume, It shows that the cloud function is running into some trouble.

And when I use ctrl + p my resume is being cut off

**Comment by DineshSolanki** [comment]
pdf exporting of both single and multipage is working fine on my side, but yes the printing does cut the content.

**Comment by SaiAnuragGudi** [comment]
what browser are you using @DineshSolanki

**Comment by DineshSolanki** [comment]
Firefox @SaiAnuragGudi

**Comment by SaiAnuragGudi** [comment]
Nope I use Brave.
Doesn't work on Firefox either @DineshSolanki

**Comment by DineshSolanki** [comment]
Well, do you mind sharing your resume link?

**Comment by SaiAnuragGudi** [comment]
https://rxresu.me/r/327pzo

**Comment by DineshSolanki** [comment]
@SaiAnuragGudi the resume seems good, I thought I would get the option to export, so anyway, try to export again and see the debug console, it might have something useful. Ctrl + shift + j (windows) should do that.

**Comment by SaiAnuragGudi** [comment]
![debug](https://user-images.githubusercontent.com/62999612/102382096-0ac02e80-3ff0-11eb-925e-3419000700c4.png)

All the console says is this @DineshSolanki

**Comment by DineshSolanki** [comment]
Can you try disabling the brave shield and then try again.

**Comment by SaiAnuragGudi** [comment]
Getting the same error :(

**Comment by DineshSolanki** [comment]
@SaiAnuragGudi thank you for all the information, I'll try to install and test on brave myself. 🙂 Will get back to you if found a solution or the issue.

**Comment by SaiAnuragGudi** [comment]
Thank you @DineshSolanki

**Comment by AmruthPillai** [comment]
@SaiAnuragGudi I did try printing it in-browser on Chrome and looked good to me, but yes, I do understand not everyone uses the Chrome Browser. As this is just a side project for me, I couldn't get myself to invest too much of time to make sure it works perfectly on all the browsers out there. (I think I just gave up on IE on the get-go). Sorry about that :( I've attached the PDF on how it looks for me.

[RxResume-327pzo.pdf](https://github.com/AmruthPillai/Reactive-Resume/files/5720985/RxResume-327pzo.pdf)
